src/components/Content/Content.test.tsx

```diff
@@ -221,18 +222,18 @@ describe("Content componenent with no buttons", () => {
 
 describe("showContent helper function", () => {
```

Comment:
When I was doing my training with Mike & Colin were keen on exhaustive tests (i.e. any test in a function would normally have a separate test). It's probably a bit ambitious retro-fitting 100% code coverage, but for these sorts small utility functions it probably make sense:

![image](https://github.com/simpleviewinc/sv-mosaic/assets/13272784/f240622f-0bf1-4712-9d78-c0f287659f3b)

Reply:
Reckon it makes sense to merge these parameterless tests into `src/utils/show/evaluateShow.test.ts` just so they're in one place?

Reply:
Introduced exhaustive tests and isolated them to the evaluateShow test suite. Review re-requested.